bin/package.json

```diff
@@ -7,6 +7,7 @@
     "inquirer": "3.0.1",
     "livescript": "1.5.0",
     "o-tools-livescript": "1.2.3",
+    "path": "0.12.7",
```

Comment:
You should be able to use the built-in [path](https://nodejs.org/api/path.html#path_path_basename_path_ext) module in Node here.